Assets/Scripts/Managers/PlanetManager.cs

```diff
@@ -10,30 +10,39 @@
 
 public class PlanetManager : MonoBehaviour
 {
-    [SerializeField]
-    private GameObject animalDisplay;
+    //private GameObject animalDisplay;
```

Comment:
Commented out line here

Reply:
I have removed this unused and commented out line.